### ISSUE TITLE
Train the model using continous loss 

### ISSUE THREAD

**Issue by explorer1212** [AlgoMole/MolCRAFT#14]
Hello,
    I want to train a model that uses continuous loss, I just let use_discrete_t=False, do I need to do anything else, it feels like the model converges at 15 epochs as well, and it doesn't converge as slowly as the paper mentions than using discrete loss
![image](https://github.com/user-attachments/assets/b26496a1-cd34-4efd-902d-18d3a156595e)

**Comment by explorer1212** [comment]
By the way, the sampled molecules have very high Strain Energy using continuous loss

**Comment by qky18** [comment]
Hi, this is very interesting and worth further exploration. After checking with the calculation of `val_loss`, I found [this line of code](https://github.com/AlgoMole/MolCRAFT/blob/master/core/callbacks/validation_callback.py#L175) actually didn't correspond to -VLB = L^n(x) + L^r(x) as Eq. 20 (Graves, et al.). To fix it, try setting it to
```python
for t in range(0, pl_module.cfg.dynamics.discrete_steps + 1, 10):
```

This calculates the reconstruction loss L^r(x) when t = 1, and will probably yield a better monitor.

By the way, what does the "high Strain Energy" actually refer to? I trained a checkpoint using continuous loss (i.e. `use_discrete_t=False`) within 15 epochs, and with `num_samples=1` the 25%, 50%, 75% of Strain Energy are around 40, 96, 312, which seems acceptable.
